src/Integration/Binding/ProjectBinderFactory.cs

```diff
@@ -47,21 +47,7 @@ internal ProjectBinderFactory(IServiceProvider serviceProvider, ILogger logger,
 
         public IProjectBinder Get(Project project)
         {
-            var languages = projectToLanguageMapper.GetAllBindingLanguagesForProject(project).ToList();
-            var isCSharpVBLanguage = languages.Contains(Core.Language.VBNET) || languages.Contains(Core.Language.CSharp);
-
-            if (isCSharpVBLanguage)
-            {
-                return new CSharpVBProjectBinder(serviceProvider, projectToLanguageMapper, fileSystem, logger);
-            }
-
-            var isCFamilyLanguage = languages.Contains(Core.Language.C) || languages.Contains(Core.Language.Cpp);
-
-            if (isCFamilyLanguage)
-            {
-                return new CFamilyProjectBinder(projectToLanguageMapper, logger, fileSystem);
-            }
-
+            // TODO - CM cleanup - remove IProjectBinderFactory
```

Comment:
We will no longer make project-level changes, so we don't need any of the project-level binding code.

There is a lot of it, so I'll remove it in stages to keep the PRs manageable.
This first PR just removes the directly implementations of `IProjectBinder` and their tests.

Next, I'll clean up the calling code i.e. get rid of `IProjectBinderFactory` and fix up the code that calls it.

Then, I'll remove the lower-level classes that are no longer used i.e. the classes that were only used by the `CSharpVBProjectBinder`.